lib/context.js

```diff
@@ -112,6 +127,13 @@ class RequestContext {
       return undefined;
     }
   }
+
+  async fetchUserInfo(opts) {
```

Comment:
my recommendation: don't expose the options. The default is the REQUIRED to implement for all providers that support the userinfo endpoint.